peachjam/models/user_following.py

```diff
@@ -160,13 +191,13 @@ def clean(self):
         if set_fields == 0:
             raise ValidationError(
                 "One of the following fields must be set: court, author, court class, court registry, country, "
-                "locality, taxonomy topic"
+                "locality, taxonomy topic saved search"
```

Comment:
you're violating DRY -- use self.follow_fields

Reply:
done